charts/ps-db/templates/cluster.yaml

```diff
@@ -416,6 +435,25 @@ spec:
       {{- end }}
       {{- if $router.gracePeriod }}
       gracePeriod: {{ $router.gracePeriod }}
```

Comment:
Missing end for gracePeriod condition

Reply:
https://github.com/percona/percona-helm-charts/pull/669/commits/89b97023919fe9bdea3acc69659b5904fb378fd8